src/platforms/javascript/common/session-replay/index.mdx

```diff
@@ -70,14 +70,14 @@ Sampling occurs when the session is first started. <PlatformIdentifier name="rep
 
 ## Error Linking
 
-Currently, errors that happen on the page while a replay is running are linked to the Replay, making it as easy as possible to jump between related issues/replays. However, please note that it is __possible__ that the error count reported on the Replay Detail page does not match the actual errors that have been captured. The reason for that is that errors can be lost, e.g. a network request fails, or similar. This should not happen to often, but be aware that it is theoretically possible.
+Currently, errors that happen on the page while a replay is running are linked to the Replay, making it as easy as possible to jump between related issues and replays. However, it is **possible** that the error count reported on the **Replays Details** page does not match the actual errors that have been captured. This is because errors can be lost; for example, a network request fails, or similar. This should not happen to often, but be aware that it is theoretically possible.
```

Comment:
```suggestion
Currently, errors that happen on the page while a replay is running are linked to the Replay, making it as easy as possible to jump between related issues and replays. However, it is **possible** that the error count reported on the **Replays Details** page does not match the actual errors that have been captured. This is because errors can be lost; for example, a network request may fail or similar. This shouldn't happen often, but it's theoretically possible.
```